### PR TITLE
Scaffold react application

### PR DESCRIPTION
**Pull request by NiciP**
#### [Scaffold React App](https://www.pivotaltracker.com/story/show/165001620)

As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.

#### Description
- Setup application
- Clean up comments

### CODE REVIEW

File: thyme.2/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (nevroje) [inline thyme.2/README.md]: Clean the entire readme, we should make our own later on.

File: thyme.2/src/serviceWorker.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
+// existing tabs open on the page have been closed, since previously cached
+// resources are updated in the background.
+
+// To learn more about the benefits of this model and instructions on how to
+// opt-in, read https://bit.ly/CRA-PWA
```
Comment (nevroje) [inline thyme.2/src/serviceWorker.js]: Clean up comments

**Comment by nevroje** [comment]
Whats up with the name "thyme.2"? Shouldn't it just be Thyme?

File: thyme.2/src/logo.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+        <circle cx="420.9" cy="296.5" r="45.7"/>
+        <path d="M520.5 78.1z"/>
+    </g>
+</svg>
```
Comment (aleciabenjamin) [inline thyme.2/src/logo.svg]: I think we can remove this, no?

File: thyme.2/src/App.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (aleciabenjamin) [inline thyme.2/src/App.css]: Don't think we need this.

**Pull request closed without merging**